options.go

```diff
@@ -0,0 +1,62 @@
+package coopdatadog
+
+import (
+	"github.com/coopnorge/go-datadog-lib/v2/errors"
+	"github.com/coopnorge/go-datadog-lib/v2/internal"
+	"github.com/coopnorge/go-logger"
+)
+
+const (
+	defaultEnableTracing        = true
+	defaultEnableProfiling      = true
+	defaultEnableExtraProfiling = false
+)
+
+// config is the internal configuration for the Datadog integration
+type config struct {
+	enableTracing        bool
+	enableProfiling      bool
+	enableExtraProfiling bool
+	errorHandler         errors.ErrorHandler
+}
```

Comment:
Would it be better to rename `config` to `options`, that way we avoid the conflict and potential confusion with the package `github.com/coopnorge/go-datadog-lib/v2/config`?

Reply:
renamed.